### DISCUSSION TITLE
How to get the default Sharepoint query suggestion provider to work?

### DISCUSSION THREAD

**Discussion by colonelclaypoo** [microsoft-search/pnp-modern-search#2197] [Q&A] [answered]
Hi all,

I was hoping that this is the only thing required to enable query suggestions:

![image](https://user-images.githubusercontent.com/29123261/177164224-c303a9e6-7675-4d39-b7ac-5d45b047632c.png)

However, upon entering a search term in the serach box the spinning wheel comes up for a split second but no suggestions appear.

I'm on SPO so not custom terms can be imported via .txt.

Thanks for your help.

**Comment by wobba** [comment, marked as answer]
Navigate to https://tenant-admin.sharepoint.com/_layouts/15/searchadmin/querysuggestionsettings.aspx?level=tenant and you can import suggestions.

**Reply by zeair** [reply]
so it requires configuration? Is there any way to configure this to work as OOTB Search Input?
  
![image](https://user-images.githubusercontent.com/17876212/178750861-f828530d-4b2a-4cbe-a7d1-0401004351d4.png)

**Reply by FranckyC** [reply]
I don't think you can no. You can try bookmarks of Microsoft Searhc but not sure they will appear in the search box as suggestions https://docs.microsoft.com/en-us/microsoftsearch/manage-bookmarks

**Comment by colonelclaypoo** [comment]
Oh right, I see. Must have missed this. Thank you very much for your help.
